torchvision/transforms/functional.py

```diff
@@ -200,10 +200,18 @@ def normalize(tensor, mean, std):
     if not _is_tensor_image(tensor):
         raise TypeError('tensor is not a torch image.')
 
-    # This is faster than using broadcasting, don't change without benchmarking
-    for t, m, s in zip(tensor, mean, std):
-        t.sub_(m).div_(s)
-    return tensor
+    if not inplace:
+        tensor_clone = tensor.clone()
+        # This is faster than using broadcasting, don't change without benchmarking
+        for t, m, s in zip(tensor_clone, mean, std):
+            t.sub_(m).div_(s)
+        return tensor_clone
+
+    else:
+        # This is faster than using broadcasting, don't change without benchmarking
+        for t, m, s in zip(tensor, mean, std):
```

Comment:
I believe this can now be replaced with something simpler and as efficient as before
```python
mean = torch.tensor(mean, dtype=torch.float32)
std = torch.tensor(std, dtype=torch.float32)
return (tensor - mean[:, None, None]) / std[:, None, None]
```
This is out-of-place, but can be made in-place as well if needed


Just to double check, can you do some benchmarkings and report back if it's indeed now giving the same runtime speed?

Reply:
@fmassa  i ran some tests and found that this approach was smewhat faster .